### ISSUE TITLE
3D Surface plot requires grid3D.postEffect.enable = true

### ISSUE THREAD

**Issue by MiltonCobo** [ecomfe/echarts-gl#382]
I notice in the Examples of the new Echarts 5 that 3D Surface is not correctly display unless grid3D.postEffect.enable is set to true.
This is not happening in the examples given for version 4.x of echarts. 

![image](https://user-images.githubusercontent.com/44674476/104145784-5dde9480-5396-11eb-9dc7-dd84b8999f3e.png)

**Comment by pissang** [comment]
It has been fixed in the next minor version:)

**Comment by MiltonCobo** [comment]
Apparently this issue continuous in the new update. Surface3D only shows if grid3D.postEffect.enable = true.
Also I notice that when I change the type to "piecewise" in the visualMap, then the surface disappears.

**Comment by pissang** [comment]
@MiltonCobo Do you mind providing your option?

**Comment by MiltonCobo** [comment]
Hi, I was just playing with your library and noticed that surfaces are not
plotted unless grid3D.postEffect.enable is set to true.
Is this the correct way? For example with this simple options:

export const Option = Object.freeze({
  visualMap: {
    show: false,
    type: "piecewise",
    splitNumber: colorPalette.length, // colorPalette = [] of colors
    dimension: 2,
    min: -28,
    max: 10,
    inverse: true,
    precision: 0,
    inRange: {
      color: colorPalette,
    },
  },
  xAxis3D: {
    type: "value",
  },
  yAxis3D: {
    type: "value",
  },

  zAxis3D: {
    type: "value",
  },
  grid3D: {
    show: true,
    boxWidth: 60,
    boxDepth: 60,
    boxHeight: 60,

    postEffect: {
      enable: false,
    },
    temporalSuperSampling: {
      enable: true,
    },

    viewControl: {
      autoRotate: false,
    },
  },
  series: [
    {
      type: "surface",
      shading: "realistic",

      equation: {
        x: {
          step: 1,
          min: 1,
          max: 250,
        },
        y: {
          step: 1,
          min: 1,
          max: 250,
        },
        z: function(x, y) {
          return 0.14 * x - 4.9 * Math.log(x) + 0.12 * y - 5.3 * Math.log(y
);
        },
      },
    },
  ],
});

The output is this:
[image: image.png]

Also I noticed a different detail. Suppose visualMap.type = piecewise. Then
If we set visualMap.min or visualMap.max in a way that the range of
z-values are not covered (for example visualMap.min=-10), then the surface
colors are not plotted. So we need to set the right values. I know this is
just a detail but I wanted to inform you.

As I told you I'm trying to build a site for my math classes (
www.dydx.ufes.br). I am a math teacher in Brazil. As I told you
I'm very excited about the new feature of plotting contour curves for
surfaces. Notice that when visualMap.type = piecewise then
the curves z = constant are naturally plotted on the surface with the color
palette. What I would like is to have a property series.contours = true in
such a way that the lines z= constant are plotted on the surface and also
the projections of these lines on the xy plane as the library plotly.js
does.

Thanks very much!,

Milton.
